modes/longitudinal/src/toolbarButtons.js

```diff
@@ -76,6 +74,68 @@ function _createSetToolActiveCommands(toolName) {
   return temp;
 }
 
+const StackImageSync = _createToggleButton(
+  'StackImageSync',
+  'link',
+  'Stack Image Sync',
+  [
+    {
+      commandName: 'toggleStackImageSync',
+    },
+  ],
+  'Enable position synchronization on stack viewports',
+  {
+    listeners: {
+      newStack: {
+        commandName: 'toggleStackImageSync',
+        commandOptions: { toggledState: true },
+      },
+    },
+  }
+);
+
+const StackPrefetch = _createToggleButton(
+  'StackPrefetch',
+  'link',
+  'Stack Prefetch',
+  [],
+  'Enable Complete Stack Prefetch',
+  {
+    listeners: {
+      newStack: {
+        commandName: 'stackPrefetch',
+      },
+    },
+    isActive: true,
+  }
+);
+
+const ReferenceLines = _createToggleButton(
+  'ReferenceLines',
+  'tool-referenceLines', // change this with the new icon
+  'Reference Lines',
+  [
+    {
+      commandName: 'toggleReferenceLines',
+      commandOptions: {},
+      context: 'CORNERSTONE',
+    },
+  ],
+  'Show reference lines',
+  {
+    listeners: {
+      activeViewportIdChanged: {
+        commandName: 'toggleReferenceLines',
+        commandOptions: { toggledState: true },
+      },
```

Comment:
there can be more than one command to run on an event, so this should be an array

Reply:
It accepts either a singleton or an array, the same as the other commands lists.